.pylintrc

```diff
@@ -0,0 +1,2 @@
+[MESSAGES CONTROL]
+disable=C0103
```

Comment:
**Comment**: Another option is to add the following on `pyproject.toml`:

```python
[tool.pylint."messages control"]
disable = "invalid-name"
``` 

However, if the problem is are just a few specific names, perhaps a better option is to include them in the list of "good names" :

```python
[tool.pylint.basic]
good-names = [ "_", "i", "n", "df"]
```

Reply:
thanks for the suggestion! (i'm assuming it's a suggestion and not mandatory?)

Reply:
Of course! I'm on "Suggesting Suzie" mode, over here!